Public/Src/FrontEnd/Script/Util/ConfigurationConverter.cs

```diff
@@ -809,6 +856,12 @@ private object ConvertAny(object valueToConvert, Type resultType, object overrid
                 return ConvertToRelativePath(overridingOrMergeIntoValue ?? valueToConvert, context);
             }
 
+            // target: UnionType

```

Comment:
>UnionType [](start = 23, length = 9)

Union or Unit?

Reply:
Ahh this is for the discriminating union you are adding... please ignore :)

---
In reply to: [293481811](https://github.com/microsoft/BuildXL/pull/462#discussion_r293481811) [](ancestors = 293481811)